app/scripts/modules/core/src/presentation/forms/layouts/ResponsiveFieldLayout.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+
+import { HelpContextProvider } from 'core/help';
+
+import { IFieldLayoutProps } from '../interface';
+
+import '../forms.less';
+
+export class ResponsiveFieldLayout extends React.Component<IFieldLayoutProps> {
+  public render() {
+    const { label, help, input, actions, validationMessage, validationStatus } = this.props;
+    const showLabel = !!label || !!help;
+
+    const helpUnder = false;
```

Comment:
is this here because we think this will be a configurable prop in the future?

Reply:
Kinda... @gcomstock's new layout has help text rendered under the input fields (instead of hidden in a tooltip) and it was easy enough to write this layout in such a way to be able to handle both.

I almost made it a prop but didn't want to open the door for inconsistency right from the start. Ideally we would be globally consistent across deck and wouldn't straddle both styles (tooltip & under), but if there is a reasonable need, we could make it configurable then.